From 4cb1169b4a43182e11b3bb656cc139cf02615ea0 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Mon, 3 Sep 2018 19:19:56 +0900
Subject: [PATCH 517/909] arm64: dts: r8a77965: Add brs option

commit 32aa0760fc25a993ff81821c4e917e31cec2b5ed from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add options to specify the number of overlays that can be
synthesized in BRS of VSPDL.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77965.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77965.dtsi b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
index 7784169e5caa..48c02e445c23 100644
--- a/arch/arm64/boot/dts/renesas/r8a77965.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77965.dtsi
@@ -1965,6 +1965,7 @@
 			power-domains = <&sysc R8A77965_PD_ALWAYS_ON>;
 			resets = <&cpg 623>;
 
+			renesas,#brs = <2>;
 			renesas,fcp = <&fcpvd0>;
 		};
 
-- 
2.17.1

